finetune/lora.py

```diff
@@ -202,6 +207,46 @@ def pad_right(x, pad_id):
     return x, y
 
 
+class LlamaDataset(Dataset):
```

Comment:
I'd avoid calling this LLaMADataset as it sounds a bit misleading to me.

```suggestion
class InstructionDataset(Dataset):
```